<?xml version="1.0" encoding="UTF-8"?>
<module external.linked.project.id=":01._What_is_an_exception-Where_is_an_exception:main" external.linked.project.path="$MODULE_DIR$/../../../../01. What is an exception/Where is an exception" external.root.project.path="$MODULE_DIR$/../../../.." external.system.id="GRADLE" external.system.module.group="19._Exceptions" external.system.module.type="sourceSet" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
  <component name="NewModuleRootManager">
    <output url="file://$MODULE_DIR$/../../../../01. What is an exception/Where is an exception/out/production/classes" />
    <exclude-output />
    <content url="file://$MODULE_DIR$/../../../../01. What is an exception/Where is an exception/src">
      <sourceFolder url="file://$MODULE_DIR$/../../../../01. What is an exception/Where is an exception/src" isTestSource="false" />
    </content>
    <orderEntry type="inheritedJdk" />
    <orderEntry type="sourceFolder" forTests="false" />
    <orderEntry type="module" module-name="19._Exceptions.util.main" />
  </component>
</module>